### PR TITLE
WIP: Making Desktop Apps more discoverable

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

**Comment by jasonblais** [comment]
I'll keep this on hold, adding a WIP label for it.

There's work in progress to update desktop app docs and bringing them to docs.mattermost.com. Once that's done, planned soon, we can merge this PR

File: source/install/desktop.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Mattermost desktop applications are available for Windows, Mac and Linux operation systems. 
+
+They support all the features of the web experience, plus: 
+
+- Auto-start in Mattermost when a user logs into their machine.
+- Connect to multiple Mattermost servers from a single interface, and switch with shortcut keys.
+- Windows: Add Mattermost to Start menu, taskbar and system tray.
+- Mac: Add Mattermsot to the applications Dock
+
+You can `download them directly <https://about.mattermost.com/downloads/>`_, or use one fo the below installation guides:
```
Comment (Razzeee) [inline source/install/desktop.rst]: Typo in `of`

File: source/install/desktop.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- `Windows Desktop installation guide <https://github.com/mattermost/desktop/blob/master/docs/setup.md#windows>`_
```
Comment (Razzeee) [inline source/install/desktop.rst]: Maybe mention the chocolatey package?

File: source/install/desktop.rst (hunk 1 above)
Comment (it33) [inline source/install/desktop.rst]: Fixed, thanks @razzeee!

File: source/install/desktop.rst (hunk 2 above)
Comment (it33) [inline source/install/desktop.rst]: Um... chocolatey just came out, propose waiting until it's got a bit more usage and feedback,

**Comment by it33** [comment]
Sounds good, I addressed feedback from @razzeee, feel free to take this over, close it, however, just a suggestion.

**Comment by it33** [comment]
@jasonblais just circling back on this. Thinking it wouldn't hurt to have a link to desktop apps in the documentation. 

It's been two weeks, so that's about 20K visits where there's no mention of desktop apps in our documentation...

**Comment by jasonblais** [comment]
Thanks @it33, makes sense.

[PR for the first version of desktop docs submitted](https://github.com/mattermost/docs/pull/503). A subsequent update for the docs planned soon, which got halted with Mattermost server v3.4 release work.

**Comment by jasonblais** [comment]
Included with PR #503, closing to track changes there

**Pull request closed without merging**